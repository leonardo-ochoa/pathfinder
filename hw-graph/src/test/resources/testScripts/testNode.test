#tests a graph can be created
#able to add nodes
#tests the nodes

# create a graph
CreateGraph A

# create a graph
CreateGraph B

# check nodes in graph A and B
ListNodes A
ListNodes B

# add nodes to A
AddNode A n1
AddNode A n2
AddNode A n3

# check nodes in graph A and B
ListNodes A
ListNodes B

# add nodes to B
AddNode B n1
AddNode B n2
AddNode B n3
AddNode B n4

# check nodes in graph A and B
ListNodes A
ListNodes B