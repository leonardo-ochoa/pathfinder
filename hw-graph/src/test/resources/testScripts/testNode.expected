#tests a graph can be created
#able to add nodes
#tests the nodes

# create a graph
created graph A

# create a graph
created graph B

# check nodes in graph A and B
A contains:
B contains:

# add nodes to A
added node n1 to A
added node n2 to A
added node n3 to A

# check nodes in graph A and B
A contains: n1 n2 n3
B contains:

# add nodes to B
added node n1 to B
added node n2 to B
added node n3 to B
added node n4 to B

# check nodes in graph A and B
A contains: n1 n2 n3
B contains: n1 n2 n3 n4